src/un1qnx/base.py

```diff
@@ -24,12 +24,12 @@ def __init__(self, *args, **kwargs):
         appier.API.__init__(self, *args, **kwargs)
         self.base_url = appier.conf("UN1QNX_BASE_URL", UN1QNX_BASE_URL)
         self.auth_url = appier.conf("UN1QNX_AUTH_URL", UN1QNX_AUTH_URL)
-        self.base_url = kwargs.get("base_url", self.base_url)
-        self.auth_url = kwargs.get("auth_url", self.auth_url)
+        self.base_url = kwargs.get("base_url", None) or self.base_url

```

Comment:
this is not standard, any reason for this one?

Reply:
Yes, the one in the description, where basically, if you try to use it like for example we do here for pusher API: https://github.com/ripe-tech/ripe-core/blob/master/src/ripe_core/models/mixins/pusherc.py#L31

We default the arguments in the methods to None, then pass None to the API client. `kwargs.get` will work successfully because the key does exist even though its value is None. Hence we get a None value instead of the wanted default.